ui/packages/consul-ui/app/components/topology-metrics/notice/default-allow.hbs

```diff
@@ -0,0 +1,23 @@
+<Notice
+  class="topology-metrics-notice-default-allow"
+  ...attributes
+  @type={{or @type "warning"}}
+as |notice|>
+    <notice.Header>
+    <h3>
+      {{t "components.consul.topology-metrics.notice.default-allow.header"}}
+    </h3>
+  </notice.Header>
+  <notice.Body>
+    <p>
+      {{t "components.consul.topology-metrics.notice.default-allow.body"}}
+    </p>
```

Comment:
Wait, that looks the same as what I have 🤔 Did you mean using the `p0` `p1` ?

Reply:
We need to be able to define paragraphs of text in the translations file, not in the template (in these cases). Ideally we'd use markdown in the translations file, but I guess we should add something later. So for now I'd put the `<p>` tags here in the translations file.

I know this seems wrong (putting HTML in the translations file), but it would make more sense once it's markdown. To try help, imagine if we had a piece of text that has 2 paragraphs, you wouldn't want to specify each paragraph as a different translation key, which means we have to somehow specify simple formatting in the translations file. Let me know if that doesn't make sense and we can go over in a bit more detail.

Reply:
In this scenario I just have 1 `<p>`. Use still want to use `default-allow.body.p0`?